dowhy/causal_refuters/partial_linear_sensitivity_analyzer.py

```diff
@@ -91,24 +111,21 @@ def get_confidence_levels(self, r2yu_tw, r2tu_w, significance_level):
         # Strength of confounding that omitted variables generate in treatment regression
         Calpha2 = r2tu_w / (1 - r2tu_w)
         Calpha = np.sqrt(Calpha2)
-        S = np.sqrt(self.S2)
+        self.S = np.sqrt(self.S2)

```

Comment:
ah you define it here. better to define it in `init` method so that every method can access it.